front50-sql/src/main/resources/db/changelog/20190530-add-is-deleted-indexes.yml

```diff
@@ -1,5 +1,9 @@
 databaseChangeLog:
```

Comment:
This existing changeset will only be applied to MySQL DB.  Adding Preconditions will not change the checksum.  Verified this.

http://www.liquibase.org/2009/03/what-effects-changeset-checksums.html